string command
string info
---
string ack